### ISSUE TITLE
[BUG] Geolocation Coordinates rather than Street Addresses (and building names) display in individual Events Information [ver. 5.0.41

### ISSUE THREAD

**Issue by knguyenjahiel** [rokwire/illinois-app#3510]
**Describe the bug**
Geolocation coordinates rather than actual street addresses and building names are displaying in individual events information. An event scheduled 11 days from today (8/8/2023) has been chosen to give developers a chance to test and reproduce the reported error.

**To Reproduce**
1. Browse >> Events Feed
2. See geolocation coordinates in individual events (please see _Fig. 1_ below)
3. Select an individual event to get detailed info. See geolocation coordinates in event detail (please see _Fig. 2_ below)

**Expected behavior**
Current prod ver. 4.3.61 correctly displays street address or building name (please see _Fig. 3_ below). 

**Screenshots**
**Fig. 1 List of events with geolocation coordinates rather than street addresses**
![5 0 41_Events_GeoLocation1 png](https://github.com/rokwire/illinois-app/assets/118769169/2630ced8-2afd-4721-aa6c-a5e3ac90649c)

**Fig. 2 Event detail with geolocation coordinate rather than street address**
![5 0 41_Events_GeoLocation2 png](https://github.com/rokwire/illinois-app/assets/118769169/2bef36f4-1f46-43e4-9195-9f74c5673898)

**Fig. 3 Correct display of street address or building number for event detail in current prod ver. 4.3.61**
![4 3 61_Events_Address png](https://github.com/rokwire/illinois-app/assets/118769169/55f6a7ee-212e-4200-b445-96764dfe444b)

**Smartphone (please complete the following information):**
 - Device: Pixel 6
 - OS: Android 13
 - Version: 5.0.41

**Comment by mihail-varbanov** [comment]
The event creators is obligated only to set geolocation coordinates for in-person events. Building name and/or address are optional. The app used to display geolocation coordinates coordinates only if there was no building name and address, and the only available location detail was its coordinated. I recently updated this behavior and if we fall in this case (to have only geolocation coordinates and no building name or address) we display nothing. This can be verifies in version 5.0.46.

**Comment by knguyenjahiel** [comment]
Update: in ver. 5.0.55, no location information is displayed for in-person events. We would like to have building name, room number, and street address displayed if they are provided.


![Screenshot (Aug 29, 2023 12_23_46 PM)](https://github.com/rokwire/illinois-app/assets/118769169/41714cb9-7faa-4cd5-a448-654f28a241ed)

**Comment by mihail-varbanov** [comment]
This is resolved in version 5.0.61*

\* Please note that we do not know street an address or room number for events unless the creator had added this information. For example, the location of "Art Gallery Reception: Mexican Muralism Exhibit from the National Autonomous University of Mexico" event is this:
```
"location": {
    "address":null,
    "building":null,
    "city":null,
    "description":"Illini Union Art Gallery",
    "floor":null,
    "full_address":null,
    "latitude":40.10921,
    "longitude":-88.22723,
    "name":null,
    "room":null,
    "state":null,
    "zip":null
}
```
We used to build location description based on building or address, now we acknowledge the "description" field as well.

**Comment by knguyenjahiel** [comment]
Confirming building name and street address are displaying in event details. Thank you, @mihail-varbanov
